tests/test_gitlab_local.py

```diff
@@ -0,0 +1,26 @@
+# MIT licensed
+# Copyright (c) 2013-2017 lilydjwg <lilydjwg@gmail.com>, et al.
+
+import os
+import pytest
+import asyncio
+pytestmark = [pytest.mark.asyncio,
+              pytest.mark.skipif("TRAVIS" in os.environ and os.environ["TRAVIS"] == "true",
+                                 reason="do not test on travis ci")]
```

Comment:
This reason is useless. Say it "rate-limited per IP". Also, you need to unset the environment variable in this test file (and restore it later).

Reply:
> you need to unset the environment variable in this test file (and restore it later).

Unset "TRAVIS" environment variable ?

Reply:
The gitlab token.

Reply:
Got it, fix is coming.